include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -277,7 +277,7 @@ class edit_distance_unbanded_global_policy :
     size_t best_score_column() const noexcept
     {
         derived_t const * self = static_cast<derived_t const *>(this);
-        return std::ranges::size(self->database) - 1u;
+        return std::ranges::size(self->database);
```

Comment:
The documentation to this function (line 276) has become wrong now!
`//!\copydoc edit_distance_unbanded_semi_global_policy::_best_score_col` documents the inclusive coordinates. 

This mistake is copydoc'ed back to semi_global's `best_score_column()` function... Aside from the intransparent chain of copydoc here: In general I think it's legitimate that an inherited class copies documentation from its base, but the opposite direction should be avoided!